gbus/bus.go

```diff
@@ -75,7 +75,8 @@ var (
 	//for a random retry time. Default is 10 but it is configurable.
 	BaseRetryDuration = 10 * time.Millisecond
 	//RPCHeaderName used to define the header in grabbit for RPC
-	RPCHeaderName = "x-grabbit-msg-rpc-id"
+	RPCHeaderName         = "x-grabbit-msg-rpc-id"
+	ResurrectedHeaderName = "x-resurrected-from-death"
```

Comment:
I am not sure we want to introduce another header for this specific scenario.
Maybe it is enough to maintain the x-death-first-exchange header rabbitmq puts in place and differentiate execution according to that header.

Reply:
We could do that for sure, personally I would rather make it more explicit -  feel like using the existance of "x-first-death-exchange" to know is less obvious to someone from the outside, and I am not sure how heavy a downside is adding a new custom header, but ultimately its a judgement call... should I change it?

Reply:
Yah, I am gravitating towards being more explicit however trying to keep the wire protocol as narrow as possible...however I am not sure that is a good idea. need to scratch my head about this a bit more

Reply:
Ok, explicitness would be better here.